src/worker.jl

```diff
@@ -39,27 +43,38 @@ function serve(server::Sockets.TCPServer)
             @debug("New connection", client_connection)
 
             # Handle request asynchronously
-            # TODO: if `begin` was `while true`, then this connection could be reused. (like in Distributed)
-            latest = @async begin
+            latest = @async while true
                 # Set network parameters, this is copied from Distributed
                 Sockets.nagle(client_connection, false)
                 Sockets.quickack(client_connection, true)
+                @static if isdefined(Base, :buffer_writes) && hasmethod(Base.buffer_writes, (IO,))
+                    Base.buffer_writes(client_connection)
+                end
```

Comment:
I think this should test for
```suggestion
                @static if isdefined(Base, :buffer_writes) && hasmethod(Base.buffer_writes, (Base.LibuvStream, Int))
                    Base.buffer_writes(client_connection, 65536 #= Base.SZ_UNBUFFERED_IO =#)
                end
```
Otherwise, this ends up calling this method which seems to do nothing?
https://github.com/JuliaLang/julia/blob/bd8ffef2b91fa12af5df2d740ab0153567e9f173/base/io.jl#L31
At least, `@less Base.buffer_writes(client_connection)` uses it whereas `Base.buffer_writes(client_connection, Base.SZ_UNBUFFERED_IO)` dispatches to [the right method](https://github.com/JuliaLang/julia/blob/d77575063d7ecb5e79ee6fc9f3b73242f8282853/base/stream.jl#L1137-L1143). This should probably be fixed upstream?

Reply:
I had played with `buffer_writes` earlier this week (https://github.com/JuliaPluto/Malt.jl/commit/aad3005b4957f02d10564128aa95798120b48879) but was actually confused by this dispatch :smile:

Reply:
Thanks! I was wondering which one we should use here.

Ah wow and you also did the message boundary, awesome, can you merge that? ☺️